src/pages/settings/settings.html

```diff
@@ -870,6 +870,23 @@
                                             </div>
                                         </td>
                                     </tr>
+                                    <tr id="companion-server-port">
+                                        <td>
+                                            <span
+                                                class="settings-item-last-child"
+                                            ></span>
+                                            <span
+                                                i18n="i18n_LABEL_SETTINGS_TAB_GENERAL_COMPANION_SERVER_PORT"
+                                            ></span>
+                                        </td>
+                                        <td>
+                                            <input
+                                                type="text"
```

Comment:
Why is this Text and not a Number?
This would greatly reduce the amount of checks we would need to do and would guarantee a number at least.

Reply:
Even if you put type="number" that does not prevent the user from inserting text.

For this reason I opted for a type="text" simply because it does not display the arrows to the right, which I would've hidden for stylistic reasons.

The port gets validated in the logic to be a number and in range